dockerfiles/cli/tests/cmd_backup_restore_tests.bats

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bats
+# Copyright (c) 2017 Codenvy, S.A.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Roman Iuvshyn
+
+source /dockerfiles/cli/tests/test_base.sh
+
+# Kill running che server instance if there is any to be able to run tests
+setup() {
+  kill_running_named_container che
+  remove_named_container che
+}
+
+teardown() {
+  docker run --rm -v "${SCRIPTS_DIR}":/scripts/base -v /var/run/docker.sock:/var/run/docker.sock -v "${tmp_path}":/data $CLI_IMAGE stop --skip:nightly --skip:pull
```

Comment:
agreed, that should be more stable. and now I;ve added separate test for `stop` command so  I will update this.